acl_upgrade.php

```diff
@@ -480,7 +483,8 @@
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'amendment', 'Amendments (write,addonly optional)', 'write');
   //Insert the 'lab' object from the 'patients' section into the Physicians group write ACL (added in 5.0.1)
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'lab', 'Lab Results (write,addonly optional)', 'write');
-
+  //Insert the 'Multipledb' object from the 'admin' section into the Administrators group write ACL (added in 5.0.1)
+  updateAcl($admin_write, 'Administrators','admin', 'Administration', 'Multipledb', 'multipledb','write');
   //DONE with upgrading to this version
   $acl_version = $upgrade_acl;
 }
```

Comment:
Look here for example by Rod on adding new stuff in the upgrade script:
(you basically need to mirror stuff you do in setup script in the upgrade script)
https://github.com/openemr/openemr/commit/e991638a2631b33fbbe79149ae74148f0cc989d2#diff-c35a6369e4d88e89e959f87e4560b213

And note you need to place your changes into a new upgrade block(number 5):
https://github.com/openemr/openemr/blob/master/acl_upgrade.php#L488-L516

And change following value to 5:
https://github.com/openemr/openemr/blob/master/version.php#L28